packages/fuels-programs/src/contract.rs

```diff
@@ -251,9 +251,18 @@ impl Contract {
     ) -> Result<CompiledContract, Error> {
         let extension = Path::new(binary_filepath).extension().unwrap();
```

Comment:
IMO it is better to avoid `unwrap` as we get very little context.
```suggestion
        let extension = Path::new(binary_filepath)
            .extension()
            .expect("Could not extract extension from file path");        
```